### ISSUE TITLE
large images appear as white tiles

### ISSUE THREAD

**Issue by piotrkan7** [lovasoa/dezoomify#57]
hi there,

I am having problem with this file using your software: https://publications-images.artic.edu/fcgi-bin/iipsrv.fcgi?deepzoom=osci/Monet_27/Stacks_Series_Composite_revised_2.ptif.dzi

chrome will instantly crash and mozilla is showing tiles as blank. is there any way to complete download?

Thank you in advice

**Comment by lovasoa** [comment]
Yes, it's a known browser bug. It should be possible to complete download with our node application. Look at our wiki.

**Comment by piotrkan7** [comment]
Thank you for your reply. Unfortunately I have no skill at all in using any of those things. Is there a easier way? Something like .exe maybe, I don't know..

**Comment by lovasoa** [comment]
I think the easiest way to do (though it's not *extremely* easy) is to
install an ubuntu VM and follow the instruction (copy and paste the
commands) at
https://github.com/lovasoa/dezoomify/wiki/Very-large-images#node-application
.

2016-06-17 21:15 GMT+02:00 piotrkan7 <notifications@github.com>:

**Comment by piotrkan7** [comment]
Well, I might actually be able to do it ;] thank you for your reply, and also - you're doing a grat job!

**Comment by lovasoa** [comment]
I think this is due to [this firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1282074). I managed to make dezoomify work on firefox with this image by changing the maximum size that firefox is allowed to allocate for a texture.

**Comment by lovasoa** [comment]
![demo_bug_dezoomify](https://cloud.githubusercontent.com/assets/552629/16342572/ee6f1fae-3a43-11e6-8fa5-05f8eafc7382.jpg)

**Comment by piotrkan7** [comment]
Thank you very much for it Sir, for fixing it. I appreciate it a lot!

**Comment by dancorpse** [comment]
After I installed node.js(directly run the downloaded application, I did not use ubuntu because I do not have any knowledge about that kind of thing), the loading images can be shown in Chrome, and but not in firefox and after I added gfx.max-alloc-size for firefox, the images shown! But finally I still fail to save the image cos my PC don't have enough memory.

**Comment by lovasoa** [comment]
I just tested again the URL you give in the issue description, and it now seems to work.

**Comment by lovasoa** [comment]
Bot-assisted investigation note:

I checked the DZI URL from the issue:

- `https://publications-images.artic.edu/fcgi-bin/iipsrv.fcgi?deepzoom=osci/Monet_27/Stacks_Series_Composite_revised_2.ptif.dzi`

The URL is valid Deep Zoom metadata, and the tile service is reachable. This issue appears to be about browser/runtime limits while assembling a very large image, not about an unsupported zoom protocol or missing metadata discovery.

I labeled this `browser problem`. I did not close it. Main uncertainty: I did not run a full browser render in this pass, but the metadata/tile checks and the existing report are consistent with a browser memory/rendering limitation.

**Comment by AlexanderDeLargeBrotherSir** [comment]
@piotrkan7, if this is resolved then please very kindly close this at your earliest convenience.
